### PR TITLE
Update paths in CONCEPTS.md to reflect full path

### PR DESCRIPTION
**Pull request by ShreeluSantosh**
As per issue #303 , I've edited the paths to ensure their full path is reflecting in CONCEPTS.md. 

I noticed that you had used the correct path in one place - line 76 - where you mentioned `/design` folder.

So using this example for uniformity throughout the markdown file, I replaced the following paths with their full correct paths:
* `_i18n` = `/code/kuusi/web/templates/tags/_i18n_`
* `web_extras.py` = `/code/kuusi/web/templatetags/web_extras.py`
* `templates` = `/code/kuusi/web/templates`
* `templates/widgets` = `/code/kuusi/web/templates/widgets`
* `template/tags` = `/code/kuusi/web/template/tags`
* `logo.scss` = `/design/scss/logo.scss`
* `custom.scss` = `/design/scss/custom.scss`
* `package.json` = `/design/package.json`

Could you verify that the full paths are correct as per this repo's structure?

Also, I'm participating in Hacktoberfest this year, so I would appreciate it if you could ensure that hacktoberfest-accepted label is added to my PR, if it's approved.

### CODE REVIEW

File: CONCEPTS.md [hunk 1]
```diff
@@ -57,25 +57,25 @@ In this case, the translations are added as-is in a JSON file:
     "BTN_MARK": "Mark this question",
 ```
 
-To use this translation value, you can use the `_i18n_` helper, e. g in a HTML template:
+To use this translation value, you can use the `/code/kuusi/web/templates/tags/_i18n_` helper, e. g in a HTML template:
```
Comment (cmllr) [inline CONCEPTS.md:60]: The helper is the function `_i18n_()` within `web_extras.py`, but there is no file called `_i18n_`

**Comment by cmllr** [comment]
Thanks for your contribution @ShreeluSantosh! Could you fix the mentioned section, all other changes are looking fine, thank you for that!

I'll tag you PR as `hacktoberfest-accepted` on merge - and as the entire repo is already tagged with the label `hacktoberfest`, your contribution should count towards hacktoberfest.  If not - please let me know so we can make it count :)

**Comment by ShreeluSantosh** [comment]
Added a commit to fix this _i18n_ issue.

**Comment by cmllr** [comment]
Thanks!

**Pull request merged**